pkgs/development/python-modules/hikari/default.nix

```diff
@@ -0,0 +1,79 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pytestCheckHook
+, pythonOlder
+, pythonRelaxDepsHook
+, pytest-runner
+, aiohttp
+, attrs
+, multidict
+, colorlog
+, pynacl
+, git
+, pytest
+, pytest-cov
+, pytest-randomly
+, pytest-asyncio
+, mock
+}:
+let
+  git_hash = "9ade5395ebf72350a869f559f7253fa429cad552";
+in
+buildPythonPackage rec {
+  pname = "hikari";
+  version = "2.0.0.dev120";
+
+  src = fetchFromGitHub {
+    owner = "hikari-py";
+    repo = "hikari";
+    rev = version;
+    hash = "sha256-+muM0uaGuOUAUO4nJ9ntIjlHSL5qPJd6M8nC9PVRAng=";
+  };
+
+  nativeBuildInputs = [
+    pythonRelaxDepsHook
+    git
+  ];
+
+  propagatedBuildInputs = [
+    aiohttp
+    attrs
+    multidict
+    colorlog
+  ];
+
+  pythonRelaxDeps = true;
+
+  passthru.optional-dependencies = {
+    server = [ pynacl ];
+  };
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    pytest-runner
+    pytest-asyncio
+    pytest-cov
+    pytest-randomly
+    mock
+  ];
+
+  pythonImportChecks = [ "hikari" ];
+
+  disabled = pythonOlder "3.7";
+
+  postPatch = ''
+    # hikari wants a Git commit hash.
```

Comment:
At least for me the comment is just describing what the postPatch is doing, whilst not giving any additional content. I personally would remove that, but if it helps people i am also okay in keeping that.

Reply:
Comment removed.